### ISSUE TITLE
Error for color function relative syntax (declaration-property-value-no-unknown)

### ISSUE THREAD

**Issue by gouch** [stylelint/stylelint-config-recommended#243]
### What steps are needed to reproduce the bug?

The new stylelint rule [introduced in v15](https://github.com/stylelint/stylelint-config-recommended/blame/451f825c6c72f57af4c0a16851dfa43a5e005970/index.js#L23) incorrectly errors for color function relative syntax:

```css
.test {
  color: color(from gray display-p3 r g b);
}
```

### What did you expect to happen?

[Valid relative syntax](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color#relative_value_syntax) for the color function should not error.

### What actually happened?

Error:
declaration-property-value-no-unknown: Unexpected unknown value "from" for property "color"

Error also shows in [csstree's validator](https://csstree.github.io/docs/validator.html), which I gather is a new upstream dependency

**Comment by gouch** [comment]
Thinking aloud: I wonder if these new “no-unknown” rules could become a repeating source of stylelint false positives. Valid CSS syntax is continually evolving, but these rules depend on a series of upstream repos playing continual catch up:

`stylelint-config-recommended` ← `css-tree` ← `mdn-data` ← valid css

In this case, the upstream color() definition in mdn-data was only updated and released [last week](https://github.com/mdn/data/releases/tag/v2.15.0) (despite being part of Baseline 2023), which might resolve this issue. But my options are to wait for a new release of css-tree that [pulls the newer mdn-data](https://github.com/csstree/csstree/blob/b3fe3e026ce131aa559427162451a60792589730/package.json#L103) or attempt a temporary override in the (to me) complicated [properties syntax](https://stylelint.io/user-guide/rules/declaration-property-value-no-unknown/#propertiessyntax--property-syntax-)
